docs/setting_up_your_development_environment.md

```diff
@@ -91,6 +100,14 @@ environment variables. `make devdata` doesn't generally need to be re-run
 unless we update the devdata repo.
 </details>
 
+<details>
+<summary>What is <code>make dev</code>?</summary>
+
+An app's `make dev` command is what starts the app. For example a web app like h will
+be running and accepting HTTP requests on its localhost port (5000 in h's case)
+when the app's `make dev` command is running.
+</details>
```

Comment:
The docs already have _What is `make services`?_ and _What is `make devdata`?_ right before this, it was weird not to also explain `make dev`.